app/frontend/src/views/Search.vue

```diff
@@ -1,7 +1,9 @@
 <template>
-  <v-container>
-    <h1 class="my-6 text-center">Search Forms (TBD)</h1>
-  </v-container>
+  <BaseSecure>
```

Comment:
why aren't we just completely removing this and the path?  seems silly to keep this as a hittable route, it should 404 not 401.  we don't have any significant source code to retain, no reason to keep this.

Reply:
Dropped the search page. Easy enough to re-introduce if/when we decide to add that type of feature for rudimentary form discovery.